scenarios/scenario-24/README.md

```diff
@@ -0,0 +1,148 @@
+## Scenario 24 - Connect to WSO2 API Manager as event hub to validate JWT subscriptions
```

Comment:
We have to mention that we need to run API manager in K8s or anywhere that has access from MG. Also mention that in this tutorial that we are using the API manager on K8s and show steps how to deploy APIM on K8s. We can point here - https://github.com/wso2/K8s-api-operator#step-4-install-the-api-portal-and-security-token-service

Reply:
Fixed by 4b0a784